mirgecom/profiling.py

```diff
@@ -51,15 +52,68 @@ class SingleCallKernelProfile:
     footprint_bytes: int
 
 
+class StatisticsAccumulator:
+    """Class that provides statistical functions for profile results.
```

Comment:
No real reason to me to make this special-purpose. Move to `mirgecom.tools`?

Reply:
Moved to `mirgecom.utils` (and pushed).